docs/howto/get-started-with-charm-testing.md

```diff
@@ -155,38 +143,32 @@ from ops import testing
 
 def test_pebble_ready_writes_config_file():
     """Test that on pebble-ready, a config file is written"""
-    harness: testing.Harness[MyCharm] = testing.Harness(MyCharm)
+    ctx = testing.Context(MyCharm)
     # If you want to mock charm config:
-    harness.update_config({'foo': 'bar'})
+    config = {'foo': 'bar'}
     # If you want to mock charm leadership:
-    harness.set_leader(True)
+    leader = True
```

Comment:
Might be more natural to move `config={'foo': 'bar'}` and `leader=True` to inline in the `State()` call.